SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -481,11 +503,107 @@ int64 USpatialActorChannel::ReplicateActor()
 	return (bWroteSomethingImportant) ? 1 : 0;	// TODO: return number of bits written (UNR-664)
 }
 
-bool USpatialActorChannel::ReplicateSubobject(UObject* Object, const FClassInfo& Info, const FReplicationFlags& RepFlags)
+void USpatialActorChannel::DynamicallyAttachSubobject(UObject* Object)
+{
+	// Find out if this is a dynamic subobject or a subobject that is already attached but is now replicated
+	FUnrealObjectRef ObjectRef = NetDriver->PackageMap->GetUnrealObjectRefFromObject(Object);
+
+	const FClassInfo* Info = nullptr;
+
+	// Subobject that's a part of the CDO by default does not need to be created.
+	if (ObjectRef.IsValid())
+	{
+		Info = &NetDriver->ClassInfoManager->GetOrCreateClassInfoByObject(Object);
+	}
+	else
+	{
+		Info = TryResolveNewDynamicSubobjectAndGetClassInfo(Object);
+
+		if (Info == nullptr)
+		{
+			// This is a faliure but there is already a log inside TryResolveNewDynamicSubbojectAndGetClassInfo
```

Comment:
I fell this should be the other way around. If the function name starts with "Try" then it implies it's allowed to fail and so shouldn't log, so the log should be here.
What happens in the case where this fails and when do we expect it to happen? Do we still end up in a well defined state? If so I'd suggest we cancel attaching the object and warn.

Reply:
True, but it's because the log would be duplicated.